.github/workflows/release.yaml

```diff
@@ -13,6 +13,17 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
+          
+      - name: Set tag output
+        id: tag
+        run: echo "tag_name=${GITHUB_REF#refs/*/}" >> "$GITHUB_OUTPUT"
+
+      - name: Install tejolote
+        uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
+
+      - run: |
+          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ steps.tag.outputs.tag_name }} github://kubernetes-sigs/karpenter/"${GITHUB_RUN_ID}" --output karpenter.intoto.json --sign
+
       - name: Create Github Release
         uses: marvinpinto/action-automatic-releases@919008cf3f741b179569b7a6fb4d8860689ab7f0 # v1.2.1
         with:
```

Comment:
Looks like we need to add this attestation file to the release or we won't be doing anything with the file that is output from the tool here: https://github.com/openvex/vexctl/blob/13fa934d15cb49ad2981ce4d3f5e6ecbef599919/.github/workflows/release.yaml#L93C11-L93C36

Reply:
@coder12git For posterity, can you attach the current output of this file for an already released tag to this PR so that we can verify the output and that there are no other changes that we need to make here ahead of the merge?

Reply:
This is the output that i got in my forked repo -
`{
  "signed": {
    "_type": "https://in-toto.io/Provenance/v0.1",
    "spec_version": "1.0.0",
    "materials": {
      "basic": {
        "type": "https://gitlab.com/attestation-example",
        "definition": {
          "uri": "git+https://gitlab.com/coder12git/karpenter.git/karpenter@v1.0.1",
          "hashes": {
            "sha256": "413da3c4330b2a047041e9a157759f58c3db99ee"
          }
        }
      }
    },
    "products": {
      "result": {
        "type": "https://gitlab.com/attestation-example",
        "definition": {
          "uri": "git+https://gitlab.com/coder12git/karpenter.git/karpenter@v1.0.1",
          "hashes": {
            "sha256": "413da3c4330b2a047041e9a157759f58c3db99ee"
          }
        }
      }
    },
    "steps": [
      {
        "name": "build",
        "expected_command": ["make"]
      },
      {
        "name": "test",
        "expected_command": ["make", "test"]
      }
    ]
  },
  "signatures": [
    {
      "keyid": "",
      "sig": "MEUCIQCa7Ijz3cZI/0zS1uXuCPNiasvFVpWnRw/YXHEAVuSPmgIgQ6mWJBN/Eh3+WErG2mT3Gf7Kt1ymBODFbmzOdyGsEAQ="
    }
  ]
}

Reply:
@jonathan-innis here is the link of that workflow which i ran in my forked repo - https://github.com/coder12git/karpenter/actions/runs/7271716887/job/19812702640